tests/qir/test_pytket_qir_14.ll

```diff
@@ -102,30 +98,24 @@ else:                                             ; preds = %entry
   br label %continue
 
 continue:                                         ; preds = %else, %then
-  %35 = call i1 @read_bit_from_reg(i64 %0, i64 0)
```

Comment:
This file is part of the tests, it is just used to compared the generated qir to what we are expecting